### PR TITLE
Move database blogs to docs

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Description

moved database integration blogs under `/node-red/database/`

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/node-red/database/dynamodb.md [hunk 1]
```diff
@@ -1,25 +1,20 @@
...
+  parent: Database
+meta:
+  title: Using DynamoDB with Node-RED
+  description: Get started with AWS' NoSQL database DynamoDB with Node-RED
+  keywords: node-red, flowfuse, integration, dynamodb, aws, nosql
+image: /node-red/core-nodes/images/node-red-dynamodb.png
 ---
 
-Amazon's DynamoDB is a fully managed NoSQL database service known for its fast and predictable performance and scalable design. This makes it suitable for applications needing low-latency responses. In this blog, we’ll look at how to set up and use DynamoDB, configure the necessary IAM roles, and apply Node-RED flows to store and retrieve data effectively.
+# Using DynamoDB with Node-RED
```
Comment (ZJvandeWeg) [inline src/node-red/database/dynamodb.md:12]: I think this would work

```suggestion
# {{ meta.title }}
```

File: src/node-red/database/index.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+eleventyNavigation:
+  key: Database
+meta:
+  title: Node-RED Database Integration Guides
+  description: Explore database integration guides for Node-RED, including PostgreSQL, MongoDB, InfluxDB, DynamoDB, and TimescaleDB
+  keywords: node-red, databases, integration, PostgreSQL, MongoDB, InfluxDB, DynamoDB, TimescaleDB
+---
+
+# Database Integration Guides
```
Comment (ZJvandeWeg) [inline src/node-red/database/index.md:10]: Does this work? 

```suggestion
# {{ meta.title }}
```

File: src/node-red/database/index.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+Welcome to the Database Integration Guides section. Here, you'll find comprehensive tutorials on integrating various databases with Node-RED. Whether you're working with SQL or NoSQL databases, or need to handle time-series data, these guides will help you connect, store, query, and visualize data effectively.
+
+- [Using PostgreSQL with Node-RED](./postgresql.md)
```
Comment (ZJvandeWeg) [inline src/node-red/database/index.md:14]: I think you can use eleventyNav to dynamically solve this.

File: src/node-red/database/influxdb.md [hunk 4]
```diff
@@ -1,17 +1,19 @@
 ---
-title: Send data from your Node-RED flows into InfluxDB
-subtitle: A Guide to Connecting InfluxDB with Node-RED
-description: Node-RED has great support for InfluxDB, in this guide we'll explain how to get your data flowing into one of the most popular time-series database.
-date: 2023-07-11
-authors: ["rob-marcer"]
-image: blog/2023/07/images/node-red-influxdb.png
+eleventyNavigation:
+  key: InfluxDB
+  parent: Database
```
Comment (ZJvandeWeg) [inline src/node-red/database/influxdb.md:4]: Should this be plural? What do you think?

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]
Real quick pass, more thorough review will follow later.

File: src/node-red/database/index.md (hunk 2 above)
Comment (sumitshinde-84) [reply src/node-red/database/index.md:10]: yes, it works!

File: src/node-red/database/influxdb.md (hunk 4 above)
Comment (sumitshinde-84) [reply src/node-red/database/influxdb.md:4]: yes, lets make it plural !

File: src/node-red/database/index.md (hunk 3 above)
Comment (sumitshinde-84) [reply src/node-red/database/index.md]: Done !

**Comment by ZJvandeWeg** [comment]
@sumitshinde-84 You've got conflicts. @Hasmin-AC Can you review before me?

**Comment by sumitshinde-84** [comment]
@ZJvandeWeg, just resolved Conflicts.

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**